src/components/common/Layout/components/Account/components/AccountModal/components/SocialAccount/EmailAddress.tsx

```diff
@@ -159,10 +159,9 @@ const ConnectEmailButton = ({
         colorScheme={emails?.pending ? "orange" : rewards.EMAIL.colorScheme}
         variant={"solid"}
         size="sm"
-        isDisabled={emails?.emailAddress}
```

Comment:
Why are these changes here? If they fix something unrelated, it should be a separate PR, but also seem like they're unintentional?

Reply:
They are unintentional. Let me look into it